### ISSUE TITLE
Digital Ocean docs for `Installing Deis`

### ISSUE THREAD

**Issue by hermanjunge** [deis/workflow#217]
Hi,

I just finished a [terraform script and docs to deploy a K8s cluster in Digital Ocean](https://github.com/hermanjunge/k8s-do-tf).

I want to fill a Pull Request for the documentation for installing Deis in DO, _but_ I'm not sure what would be the scope of this documentation:

Do we start from zero? or Do we start from `have a functional k8s cluster in DO and then...`.

**Comment by slack** [comment]
Excellent! Looking forward to trying out your stuff!

Currently leaning toward "have a functional k8s cluster," now what... We bounced around a few ideas here https://github.com/deis/workflow/pull/163#discussion-diff-48198801

Instead of documenting all the steps required to go from zero to kubernetes, we are planning on linking out to external resources which walk folks through k8s standup. I think linking your DO instructions would nicely fit there.

We've got a ton of un-merged documentation changes at the moment. As soon as the dust settles there should be an obvious spot to contribute.
